tests/usbredir_test.go

```diff
@@ -175,10 +175,27 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 	})
 })
 
-func createVMI(virtClient kubecli.KubevirtClient, enableUsbredir bool) (*v1.VirtualMachineInstance, error) {
-	randomVmi := tests.NewRandomVMI()
-	if enableUsbredir {
-		randomVmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
+func withClientPassthrough() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
 	}
-	return virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVmi)
+}
+
+func createVMIWithUsbredirEnabled(virtClient kubecli.KubevirtClient) *v1.VirtualMachineInstance {
+	randomVmi := libvmi.New(
```

Comment:
there's nothing random about it. The name `vmi` would befit it better.
But more importantly, I think that this entire function can be inlined into its single user. `libvmi` makes it very clear to understand what is being built, so the function does not add much helpful abstraction.

Reply:
Right probably leftovers from the old code 
will update soon :)

Reply:
Done